hr_expense_advance_clearing_sequence/hooks.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import SUPERUSER_ID, api
+
+
+def assign_old_sequences(cr, registry):
```

Comment:
Why are we overwriting the new number to already assigned number?
I think we shouldn't change the existing number, should we?

Reply:
because advance document should number AVEX instead of EX, or not?
but EX sequence will skip this number. I'm not sure what is best way, Could you suggest?

Reply:
First small thing. I think the number should be AVxxxx (not AVEX)
I am not sure when you say, EX will skip the number?

Reply:
Okay, I see.